unime/src-tauri/tests/tests/utils.rs

```diff
@@ -69,7 +69,7 @@ async fn when_mime_type_is_supported_then_download_should_start() {
 }
 
 #[tokio::test]
-async fn when_mime_type_is_not_supported_then_download_should_fail() {
+async fn when_media_type_is_not_supported_then_download_should_fail() {
```

Comment:
This test is failing now because it still expects 0 triggers (`.expect(0)`), because previously there was no request to this endpoint at all, but now there is.